articles/libraries/lock/v11/configuration.md

```diff
@@ -768,7 +768,8 @@ Allows to set the initial value for the email and/or username inputs. When omitt
 var options = {
   prefill: {
     email: "someone@auth0.com",
-    username: "someone"
+    username: "someone",
+    phoneNumber: "+1234567890"

```

Comment:
For what it's worth that's not a valid phone number.  Maybe +1 555 234 5678 ?

Reply:
I think people will understand it's just an example 😬